rack-box/README.md

```diff
@@ -40,6 +40,10 @@ although we will mention each file here as well:
   <https://jena.apache.org/download/> and rename it (note we still are
   using version 3.16.0 instead of the latest release, though)
 
+- `files/apache-jena.tar.gz`: On the same page you downloaded Fuseki, scroll
```

Comment:
We've changed lines 39-41 in the master branch.  You should update lines 43-45 to match as well:

```
- `files/jena.tar.gz`: Download latest Jena release tarball from
  <https://jena.apache.org/download/> and rename it
```

Reply:
You still need to update these lines.